sidekick_core/lib/src/update_sidekick_cli.dart

```diff
@@ -99,9 +101,9 @@ class UpdateToolsMigration extends MigrationStep {
   Future<void> migrate(MigrationContext context) async {
     final template = SidekickTemplate();
     final props = SidekickTemplateProperties(
-      name: Repository.requiredSidekickPackage.cliName,
-      entrypointLocation: Repository.requiredEntryPoint,
-      packageLocation: Repository.requiredCliPackage,
+      name: SidekickContext.sidekickPackage.cliName,
+      entrypointLocation: SidekickContext.entryPoint,
+      packageLocation: SidekickContext.sidekickPackage.root,
```

Comment:
Can we access `SidekickContext` statically in the templates and not forward every property? Seems redundant

(for a later investigation)